pkg/ext/certs.go

```diff
@@ -236,9 +236,10 @@ func (p *rotatingSNIProvider) Run(stopChan <-chan struct{}) error {
 			}
 		case event, ok := <-watcher.ResultChan():
 			if !ok || event.Type == watch.Error {
-				logrus.Errorf("watcher channel closed: %v", err)
+				logrus.Errorf("watcher channel closed: %v", event.Object)
```

Comment:
Do you have an example of before vs after with this change? Wondering how the message looks like.